web/portal/src/main/webapp/WEB-INF/conf/organization/idm-configuration.xml

```diff
@@ -163,12 +177,13 @@
           <field name="groupTypeMappings">
             <map type="java.util.HashMap">
               <entry>
-                <key>
-                  <string>/</string>
-                </key>
-                <value>
-                  <string>root_type</string>
-                </value>
+                <key><string>/</string></key>
+                <value><string>root_type</string></value>
+              </entry>
+
+              <entry>
```

Comment:
Is there a way to make this configured from external too ? I mean enabling group configuration or not ? and deciding to which group they will be mapped ?

Reply:
I think so, I will check that. By default I want to map the groups at the root level (see TODO list), but I did not succeed so far.

Reply:
OK